application/src/main/java/dev/stratospheric/todoapp/config/AmazonDynamoDBConfig.java

```diff
@@ -5,13 +5,21 @@
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapperConfig;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
 
 @Configuration
 public class AmazonDynamoDBConfig {
 
   @Bean
-  public DynamoDBMapper dynamoDBMapper(AmazonDynamoDB amazonDynamoDB) {
-    return new DynamoDBMapper(amazonDynamoDB, DynamoDBMapperConfig.DEFAULT);
+  public DynamoDBMapper dynamoDBMapper(AmazonDynamoDB amazonDynamoDB, Environment environment) {
+
+    String environmentName = environment.getProperty("custom.environment");
+    String applicationName = environment.getProperty("spring.application.name");
+
+    return new DynamoDBMapper(amazonDynamoDB, DynamoDBMapperConfig.builder()
+      .withTableNameOverride(DynamoDBMapperConfig.TableNameOverride
```

Comment:
this lets us have tables per stage. As of now, we only have the table `breadcrumbs` which will clash as soon as we would deploy a second environment.

This neat feature let's us dynamically prefix our table names with e.g. `staging-todo-app-mytablename`